Assessments/Template.md

```diff
@@ -23,6 +24,12 @@
 |• Code base is compatible with search engines |   |
 |**Ease of use**|  |
 |• Programming language and existing code base and extensions are easy to use |   |
+|**Performance**|   |
+|• Tool's performance is competitive with similar commercial and open source tools |   |
+|**Reliability**|   |
+|• Tool's reliability is competitive with similar commercial and open source tools |   |
+|**Scaleability**|   |
+|• Tool's scaleability is competitive with similar commercial and open source tools |   |
 
```

Comment:
Resolved merge conflict and changed "equivalent or better than" to "competitive with" since intent is to flag big differences in performance, reliability and scaleability rather than small ones.